dashboard/v1.1/src/pages/Monitoring/Matrix.tsx

```diff
@@ -195,7 +195,19 @@ const Element: FC<ElementProps> = ({ timeSeriesPath, showRouteDetails }) => {
         </Badge>
       </TableCell>
       <TableCell style={{ minWidth: 170, fontSize: 16 }} align="center">
-        <Badge color="success">{'UP'}</Badge>
+        {data.monitor === undefined ? (
```

Comment:
I think, `data.monitor === undefined` can be replace by `!data.monitor`. Can you please confirm this?